report_thesis/src/sections/background.tex

```diff
@@ -148,6 +150,103 @@ \subsubsection{Partial Least Squares (PLS)}
 The components are then used to predict the target variable by fitting a linear model via least squares regression.
 
 \subsubsection{Support Vector Regression (SVR)}
+\gls{svr} is a regression technique that extends the principles of \gls{svm} to regression problems.
+We therefore provide an overview of \gls{svm}s based on \citet{James2023AnIS} before discussing \gls{svr}s.
+
+\gls{svm} is a supervised learning algorithm used primarily for classification tasks.
+A core concept in \gls{svm} is the \textit{hyperplane}.
+Generally, a hyperplane is a subspace of one dimension less than its ambient space.
+This means that in a two-dimensional space, a hyperplane is a line, while in a three-dimensional space, it is a plane, and so on.
+
+\gls{svm} is built on the idea of finding the hyperplane that best separates the data points into different classes.
+This hyperplane is chosen to maximize the margin, which is the distance between the hyperplane and the nearest data point from either class.
+The data points that lie closest to the hyperplane are called \textit{support vectors}.
+
+% Mathematically, \gls{svm} solves the optimization problem:
+
+% $$
+% \min_{\mathbf{w}, b, \xi} \frac{1}{2} \|\mathbf{w}\|^2 + C \sum_{i=1}^{n} \xi_i,
+% $$
+
+% subject to:
+
+% $$
+% y_i (\mathbf{w} \cdot \mathbf{x}_i + b) \geq 1 - \xi_i,
+% $$
+
+% where:
+
+% \begin{itemize}
+% 	\item $\mathbf{w}$ is the normal vector to the hyperplane,
+% 	\item $b$ is the bias term,
+% 	\item $\xi_i$ are slack variables that allow some points to be within the margin or misclassified, and
+% 	\item $C$ is a regularization parameter balancing the trade-off between maximizing the margin and minimizing the classification error.
+% \end{itemize}
+
+\gls{svr} extends the principles of \gls{svm} to regression problems.
+We use our previous discussion of \gls{svm} to introduce \gls{svr} based on \citet{druckerSVR}.
+
+Instead of finding the hyperplane that best separates data points, \gls{svr} aims to fit a function that predicts continuous values.
+But instead of using a hyperplane to separate the data, \gls{svr} uses two parallel hyperplanes to define a margin within which the function should lie, often referred to as the \textit{tube}.
```

Comment:
From my understanding, it's actually often referred to as the epsilon tube (because $\epsilon$ is the hyperparameter controlling the 'width' covered by the two hyperplanes). But fine to refer to tube.

You write `instead of` twice, kind of referring to the same thing. I just reformulated to remove that.
```suggestion
\gls{svr} aims to fit a function that predicts continuous values rather than finding the hyperplane that best separates data points. 
Instead of using a hyperplane to separate the data, \gls{svr} uses two parallel hyperplanes to define a margin within which the function should lie, often referred to as the \textit{tube}.
```

Reply:
I added a little bit to clarify the part about the tube width with epsilon.